src/core/dfn.js

```diff
@@ -71,11 +71,11 @@ export function run() {
     computeType(dfn, linkingText);
     computeExport(dfn);
 
-    // Only add `lt`s that are different from the text content
-    if (titles.length === 1 && linkingText === norm(dfn.textContent)) {
-      continue;
+    // Only add `lt`s that are different from the text content and local-lts
+    const lt = titles.filter(t => !((dfn.dataset.localLt || "").split("|").includes(t)));
```

Comment:
Might be a good idea to add a `const localLt` to avoid splitting on each `filter` callback.

Reply:
done in https://github.com/speced/respec/pull/5014/commits/ae73f00da97d1c5dd11a7637ed7937078274c320